src/ipc.ts

```diff
@@ -186,6 +187,11 @@ ipcMain.on("ipcCall", async function (_ev: IpcMainEvent, payload) {
                 thumbnailURL: source.thumbnail.toDataURL(),
             }));
             break;
+        case "callDisplayMediaCallback":
+            await getDisplayMediaCallback()?.({ video: args[0] });
+            setDisplayMediaCallback(null);
```

Comment:
Not a fan of this but don't have suggestions to improve it :D

Reply:
Hah, yeah, I also managed to disgust myself but wasn't sure how to fix it. (that could also described me making my first pizza)